compiler/rustc_codegen_ssa/src/back/link.rs

```diff
@@ -1046,7 +1046,15 @@ fn link_natively(
     let strip = sess.opts.cg.strip;
 
     if sess.target.is_like_osx {
-        let stripcmd = "rust-objcopy";
+        let stripcmd = if sess.target.os == "ios" {
```

Comment:
I'd prefer something like:
```suggestion
        let stripcmd = if cfg!(target_os = "macos") && sess.target.os != "macos" {
```

To not further break cross-compilation from Linux->iOS (which is rare, but possible), and to also fix this on tvOS, watchOS and visionOS.

Reply:
Makes sense.

Reply:
Fixed.